### PR TITLE
Feat: Add enums blogs

### PR DESCRIPTION
**Pull request by adityaoberai**
## What does this PR do?

Adds init enums blogs:
- Enhancing type safety in software development with enums
- Effective use of enums in API design

## Test Plan

- `pnpm i && pnpm run dev`

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by adityaoberai** [comment]
@LauraDuRy the product announcement and docs links need to be added

**Review by LauraDuRy — APPROVED** [review]

File: src/routes/blog/post/enhancing-type-safety/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const account = new Account(client);
+
+account.createOAuth2Session(OAuthProvider.Apple, '[LINK_ON_SUCCESS]', '[LINK_ON_FAILURE]', true);
+```
+
+# Resources
+
+Visit our documentation to learn more about Enums SDK support, join us on Discord to be part of the discussion, view our blog and YouTube channel, or visit our GitHub repository to see our open-source code.
+
+- [Docs](https://appwrite.io/docs)
```
Comment (adityaoberai) [inline src/routes/blog/post/enhancing-type-safety/+page.markdoc:84]:
```suggestion
- [Docs](https://appwrite.io/docs/sdks#enums)
```

File: src/routes/blog/post/enums-api-design/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+const account = new Account(client);
+
+account.createOAuth2Session(OAuthProvider.Apple, '[LINK_ON_SUCCESS]', '[LINK_ON_FAILURE]', true);
+```
+
+# Resources
+
+Visit our documentation to learn more about Enums SDK support, join us on Discord to be part of the discussion, view our blog and YouTube channel, or visit our GitHub repository to see our open-source code.
+
+- [Docs](https://appwrite.io/docs)
```
Comment (adityaoberai) [inline src/routes/blog/post/enums-api-design/+page.markdoc:93]:
```suggestion
- [Docs](https://appwrite.io/docs/sdks#enums)
```

**Review by adityaoberai — COMMENTED** [review]
@LauraDuRy it would be good to link this here too :)

**Review by LauraDuRy — APPROVED** [review]

**Pull request merged**